homeworks/vasyl.biliak_VasylBiliak/GamesProject/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, serif;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.main {
+  height: 100%;
+  min-height: calc(100vh - 40px);
+}
+
+.navbar {
+  background-color: #EF4934;
+  display: flex;
+  align-items: center;
+  padding-left: 30px;
+  height: 80px;
+}
+
+.content {
+  display: flex;
+  margin: 60px 53px;
+}
+
+.logo_img {
+  width: 55px;
+  height: 57px;
+  margin-right: 30px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.about_me_img {
+  width: 532px;
```

Comment:
pls read more about BEM methodology. It's not so hard use correct  names for classes.